test/test_va_api_fixture.h

```diff
@@ -97,6 +100,28 @@ class VAAPIFixture : public ::testing::Test
         // before the destructor).
     }
 
+    static void SetUpTestSuite() {
+        if (m_sharedDrmHandle < 0) {
+            m_sharedVaDisplay = getDrmDisplay(m_sharedDrmHandle);
+
+            int majorVersion, minorVersion;
+            m_sharedInitStatus = vaInitialize(m_sharedVaDisplay, &majorVersion, &minorVersion);
+        }
+    }
+
+    static void TearDownTestSuite() {
+        if (m_sharedDrmHandle >= 0) {
+            if (m_sharedInitStatus == VA_STATUS_SUCCESS) {
+                vaTerminate(m_sharedVaDisplay);
+                m_sharedVaDisplay = nullptr;
+            } else {
+                m_sharedInitStatus = VA_STATUS_SUCCESS;
+            }
+
+            close(m_sharedDrmHandle);
+            m_sharedDrmHandle = -1;
+        }
+    }
```

Comment:
These should only be declared in the header file and defined in the definition (cpp) file.

Reply:
OK, I made the class, and also since the initialize/terminate behavior should be the same for any user of the class, I moved that logic to the SetUp/TearDown of it so that the consumers don't need to replicate it.